src/administrator/includes/rbacl.php

```diff
@@ -199,4 +201,65 @@ public static function getRoleByUser($userId, $client = '', $clientContentIid =
 
 		return $roles;
 	}
+
+	/**
+	 * Method to Get roles of users again to selected agency.
+	 *
+	 * @param   integer  $contentId  agency id
+	 * @param   integer  $userId    user id
+	 * @param   integer  $roleId    selected role id
+	 *
+	 * @return 	mixed
+	 *
+	 * @since 1.6
```

Comment:
@sanjivani1812 replace since tag to constant so it will get replaced automatically